chainer/training/extensions/parameter_statistics.py

```diff
@@ -103,15 +106,23 @@ def __call__(self, trainer):
                         # since the statistics function should make no
                         # assumption about the axes
                         params = getattr(param, attr_name).ravel()
-                        value = function(params)
+                        if self.check_nan_params and \
+                                cuda.get_array_module(params).isnan(params) \
+                                    .any():
```

Comment:
Avoid backslashes for line continuation. Use parentheses instead.
Also, place `and` at the beginning of the next line.
(See PEP-8).

Reply:
Thank you, fixed!